### ISSUE TITLE
onmousedown vs onclick

### ISSUE THREAD

**Issue by loteoo** [loteoo/hyperapp-site-generator#2]
I personally enjoy onmousedown navigation. It obviously feels snappier, but I'm afraid I'm missing some accessibility convention or something.

Should use onclick instead simply because that's what people expect?

**Comment by loteoo** [comment]
I'm going for onclick for convention
